pkg/generators/models/templates/model.gotpl

```diff
@@ -50,6 +50,7 @@ func (m {{$modelName}}) Validate() error {
 					{{- if .IsIPv4 }}is.IPv4,{{ end }}
 					{{- if .IsIPv6 }}is.IPv6,{{ end }}
 					{{- if .IsIP }}is.IP,{{ end }}
+					{{- if .Pattern }}validation.Match(regexp.MustCompile(`{{ .Pattern }}`)),{{ end }}
```

Comment:
Can we move the regexp compile step out of this function? Perhaps as a global private variable with a constructed name, so we don't have to compile it every time

Reply:
done